web/apps/serve.go

```diff
@@ -133,6 +133,12 @@ func ServeAppFile(c echo.Context, i *instance.Instance, fs AppFileServer, app *a
 		return err
 	}
 	defer content.Close()
+
+	ext := path.Ext(file)
+	if ext != ".html" && ext != ".htm" {
+		return fs.ServeFileContent(c.Response(), c.Request(), modtime, slug, route.Folder, file)
+	}
```

Comment:
Why are you doing this? We already have a check a few lines before to use templates only for the index file of a route. Do you see a case where this index file won't be HTML?